driver-core/src/main/java/com/datastax/driver/core/querybuilder/Insert.java

```diff
@@ -139,7 +139,7 @@ public Options using(Using using) {
      * Sets the 'IF NOT EXISTS' option for this INSERT statement.
      * <p>
      * An insert with that option will not succeed unless the row does not
-     * exist at the time the insertion is execution. The existence check and
+     * exist at the time the insertion is executing. The existence check and
```

Comment:
I'm nitpicking but for me the correct sentence would be "...at the time the insertion is _executed_".